examples/crypto-trading-basic.ipynb

```diff
@@ -328,6 +328,16 @@
     "### Positions"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Hot fix for '/' not being supported for position calls\n",
```

Comment:
Good catch! Thank you!!!
Actually this is not hotfix but we need to pass coin name for position instead of trading coin pairs.
Therefore, could you please remove `Hot fix` from the comment?
How about below?
```suggestion
    "# '/' not being supported for position calls\n",
```